codegen/src/main/ops/org/nd4j/codegen/ops/RNN.kt

```diff
@@ -100,7 +197,28 @@ fun SDRNN() =  Namespace("SDRNN") {
 
         Doc(Language.ANY, DocScope.ALL) {
             """
-             The LSTM layer.  Does multiple time steps.
+             The LSTM block
+            """.trimIndent()
+        }
+    }
+
+
+
+    Op("lstmlayer") {
+        javaPackage = namespaceJavaPackage
+        javaOpClass = "LSTMLayer"
+        Input(NUMERIC, "x") { description = " Input, with shape dependent on the data format (in config)." }
+        Input(NUMERIC, "cLast") { description = "Previous/initial cell state, with shape [batchSize, numUnits]" }
+        Input(NUMERIC, "yLast") { description = "Previous/initial cell output, with shape [batchSize, numUnits]" }
+        Input(NUMERIC, "maxTSLength")
+        useConfig(LSTMLayerWeights)
+        useConfig(LSTMLayerConfig)
+
+        Output(NUMERIC, "output") { description = "The layer's outputs." }
+
+        Doc(Language.ANY, DocScope.ALL) {
+            """
+             The LSTM layer
```

Comment:
Let's add a much better description here - i.e., mention the many different configuration it supports (fwd/bwd/bidirectional, gate configuration, different data formats etc etc)
Also mention that it supports MKLDNN (DNNL) and cuDNN acceleration.